src/Hamiltonians/Molecular.jl

```diff
@@ -3,14 +3,20 @@
     
     MolecularMolecularHamiltonian(fd::ElemCo.FciDumps.FDump[, starting_address::FermiFS2C])
 
-Implements a Molecular Hamiltonian with [ElemCo.jl](https://elem.co.il). 
+Implements an electronic ab-initio Hamiltonian based on electron overlap integrals in
+FCIDUMP format. It can be used to describe the electronic structure of a molecule
+with fixed nuclear positions (i.e. under the Born-Oppenheimer approximation).
+The parse of FCIDUMP file depends on [ElemCo.jl](https://elem.co.il). 
 
 The expression of Hamiltonian under Bohn-Oppenheimer approximation is 
 ```math
 \\begin{aligned}
     \\hat{H}   & = \\hat{H}_0 + \\hat{H}_1 + \\hat{H}_2, \\\\
```

Comment:
```suggestion

The expression for the Hamiltonian is 
```math
\\begin{aligned}
    Ĥ   & = h₀ + Ĥ₁ + Ĥ₂, \\\\
```